core/dotnet3.0/CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# .NET Core 3.0 OpenWhisk Runtime Container
+
```

Comment:
there's no 3.0 release yet. So I think you should remove the below.

@dgrove-oss ?

Reply:
I think leaving a stub in that 1.13 is the initial release is fine.  Sorry to be slow; didn't realize you were waiting for me...

Reply:
ohh, I see.  Yes 1.14 would be the first release with .NET 3.0 core.  So the 1.13 should be removed here and a 1.14 with initial release added.